com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/handlers/v2/DescriptorParser.java

```diff
@@ -26,19 +23,6 @@ protected DescriptorParser(SchemaValidator mtadValidator, SchemaValidator mtaext
         this.mtaextValidator = mtaextValidator;
     }
 
-    public ExtensionDescriptor parseExtensionDescriptorYaml(InputStream yaml) throws ParsingException {

```

Comment:
Why remove these?

Reply:
Because further configuration is needed and further refactoring. And these methods are used only in unit tests.